plugins/out_datadog/datadog_remap.c

```diff
@@ -0,0 +1,108 @@
+/* -*- Mode: C; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- */
+
+/*  Fluent Bit
+ *  ==========
+ *  Copyright (C) 2019      The Fluent Bit Authors
+ *  Copyright (C) 2015-2018 Treasure Data Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+#include <fluent-bit/flb_info.h>
+#include <fluent-bit/flb_utils.h>
+#include <fluent-bit/flb_pack.h>
+
+#include "datadog.h"
+#include "datadog_remap.h"
+
+const char* ARN_PREFIX = "arn:aws:ecs:";
+
+static void dd_remap_append_kv_to_ddtags(const char* key, const char* val, size_t val_len, flb_sds_t dd_tags)
+{
+        flb_sds_cat(dd_tags, key, strlen(key));
+        flb_sds_cat(dd_tags, ":", 1);
+        flb_sds_cat(dd_tags, val, val_len);
+        flb_sds_cat(dd_tags, ",", 1);
```

Comment:
nits, do we want to append `,` first if `dd_tags length > 0` ?

Reply:
Yes. I was thinking about saving a if-branching on every append, but your way makes code look cleaner. So I changed